### PR TITLE
Adding disaster picker to navbar

### PR DESCRIPTION
**Pull request by RamV13**

### CODE REVIEW

**Comment by janakdr** [comment]
I think Julie is better qualified than me to think about how the disaster picker will interact with the other disaster pages.

File: docs/disaster_picker.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div>
+  <select id="disaster-dropdown"></select>
+</div>
```
Comment (janakdr) [inline docs/disaster_picker.html]: nit: newline

File: docs/disaster_picker.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import {getDisaster, getDisasters} from './resources.js';
+
+export {initializeDisasterPicker};
+
+/**
+ * Initializes the disaster picker.
```
Comment (janakdr) [inline docs/disaster_picker.js]: nit, opt: if you want, can one-line this doc

File: docs/script.js [hunk 3]
```diff
@@ -66,5 +68,6 @@ function setup() {
 
 setup();
 
-// TODO(ramvellanki): switch to disaster picker
-loadNavbar('Houston Damage/Poverty Map');
+loadNavbar(() => {
```
Comment (janakdr) [inline docs/script.js]: Nit: remove braces

**Comment by RamV13** [comment]
Added styling and tests. PTAL whenever you get a chance. Thanks!

File: docs/disaster_picker.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    disasterItem.innerHTML = disaster;
+    if (disaster === getDisaster()) {
+      disasterItem.selected = 'selected';
+    }
+    disasterDropdown.appendChild(disasterItem);
+  });
+
+  disasterDropdown.onchange = () => {
+    const params = new URLSearchParams(window.location.search);
+    params.set('disaster', disasterDropdown.value);
```
Comment (janakdr) [inline docs/disaster_picker.js]: why not use localStorage here too, once the other PR is merged in here?

Comment (RamV13) [reply docs/disaster_picker.js]: will do

Comment (RamV13) [reply docs/disaster_picker.js]: fixed in child PR

**Pull request merged**